command/context/list.go

```diff
@@ -0,0 +1,50 @@
+package context
+
+import (
+	"fmt"
+
+	"github.com/smallstep/cli/flags"
+	"github.com/urfave/cli"
+	"go.step.sm/cli-utils/command"
+	"go.step.sm/cli-utils/step"
+)
+
+func listCommand() cli.Command {
+	return cli.Command{
+		Name:      "list",
+		Usage:     "list available certificate authority contexts",
+		UsageText: "**step context list**",
+		Description: `**step context list** command lists available certificate authority contexts.
+
+## EXAMPLES
+
+List all certificate authority contexts:
+'''
+$ step context list
+▶ alpha-one
+alpha-two
+ssh.beta
+'''`,
+		Action: command.ActionFunc(listAction),
+		Flags: []cli.Flag{
+			flags.HiddenNoContext,
```

Comment:
If you don't have a context selected (for whatever reason) and you run `step context [list|remove|add|etc] without the hidden flag it would prompt you to select a context, which isn't necessary for that command group.